perception/autoware_detected_object_validation/src/lanelet_filter/lanelet_filter.cpp

```diff
@@ -128,16 +140,20 @@ void ObjectLaneletFilterNode::objectCallback(
 bool ObjectLaneletFilterNode::filterObject(
   const autoware_perception_msgs::msg::DetectedObject & transformed_object,
   const autoware_perception_msgs::msg::DetectedObject & input_object,
-  const lanelet::ConstLanelets & intersected_lanelets,
+  const bgi::rtree<BoxAndLanelet, RtreeAlgo> & local_rtree,
   autoware_perception_msgs::msg::DetectedObjects & output_object_msg)
 {
   const auto & label = transformed_object.classification.front().label;
   if (filter_target_.isTarget(label)) {
+    // no tree, then no intersection
+    if (local_rtree.size() == 0) {
```

Comment:
How about this?
```suggestion
    if (local_rtree.empty()) {
```
https://beta.boost.org/doc/libs/1_68_0/libs/geometry/doc/html/geometry/reference/spatial_indexes/boost__geometry__index__rtree/empty__.html

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/8240/commits/6997be6d168cef758f8ca4bb1c0d334ebab661ec